python/lsst/meas/extensions/scarlet/scarletDeblendTask.py

```diff
@@ -592,13 +592,21 @@ class ScarletDeblendConfig(pexConfig.Config):
 
     # Size restrictions
     maxNumberOfPeaks = pexConfig.Field(
-        dtype=int, default=0,
+        dtype=int, default=200,
         doc=("Only deblend the brightest maxNumberOfPeaks peaks in the parent"
              " (<= 0: unlimited)"))
     maxFootprintArea = pexConfig.Field(
-        dtype=int, default=1000000,
+        dtype=int, default=int(1e5),
```

Comment:
For large integers like this, it's convenient to take advantage of the fact that `_`s are allowed in python ints.  So this can be `100_000` which I think is easier to read than either of the above options.

Reply:
TIL that is allowed!